index.bs

```diff
@@ -2992,6 +3031,21 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member MUST be
 
         See {{UserVerificationRequirement}} for the description of {{AuthenticatorSelectionCriteria/userVerification}}'s values and semantics.
 
+    :   <dfn>attestation</dfn>
+    ::  The [=[RP]=] MAY use this OPTIONAL member to specify a preference regarding [=attestation conveyance=].
+        Its value SHOULD be a member of {{AttestationConveyancePreference}}.
+        [=Client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=].
+
+        The default value is {{AttestationConveyancePreference/none}}.
+
+    :   <dfn>attestationFormats</dfn>
+    ::  The [=[RP]=] MAY use this OPTIONAL member to specify a preference regarding the [=attestation=] statement format used by the [=authenticator=].
+        Values SHOULD be taken from the IANA "WebAuthn Attestation Statement Format Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
+        Values are ordered from most preferable to least preferable.
+        This parameter is advisory and the [=authenticator=] MAY use an attestation statement not enumerated in this parameter.
```

Comment:
I must have missed something - when did this attestationFormats parameter get added as an optional RP-supplied property, and what is the use case driving that change to the spec?